webapp/components/wpt-amend-metadata.js

```diff
@@ -353,7 +367,38 @@ class AmendMetadata extends LoadingState(PathInfo(ProductInfo(PolymerElement)))
         node['label'] = '';
       }
       this.displayedMetadata.push(node);
+      this.fieldsFilled.filled.push(false);
     }
+    // A URL or label must be supplied for every triage item,
+    // which are all currently empty.
+    this.fieldsFilled.numEmpty = this.displayedMetadata.length;
+  }
+
+  handleFieldInput(event) {
+    // Detect which input was filled.
+    const index = event.model.__data.index;
+    const url = this.displayedMetadata[index].url;
+    const label = this.displayedMetadata[index].label;
+
+    // Check if the input is empty.
+    if (url === '' && (label === '' || label === undefined)) {
+      // If the field was previously considered filled, it's now empty.
+      if (this.fieldsFilled.filled[index]) {
+        this.fieldsFilled.numEmpty++;
+      }
+      this.fieldsFilled.filled[index] = false;
+    } else if (!this.fieldsFilled.filled[index]) {
```

Comment:
Optional nit: I would explicitly check this.fieldsFilled.filled[index] is false here because `!undefined` `!null` is also true in Javascript